### PR TITLE
Add client validation

### PR DESCRIPTION
**Pull request by Eldonratzlaff**
Client-side form validation

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @Eldonratzlaff,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has  no contribution from your pair programming partner. Kindly check the[ work and submission mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#work-and-submission-mode) of this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Eldonratzlaff** [comment]
Hello
My coding partner Abu bakar left the group on Tuesday.
And my other coding partner stopped appearing too, so I'm finishing my projects alone.

**Comment by huseyinbiyik** [comment]
Hello @Eldonratzlaff, unfortunately, we need to see an email from Student Success that shows you can work alone on your projects. If you have one please share.

**Comment by Eldonratzlaff** [comment]
Hello.
They still don't answer me.

File: index.html [hunk 1]
```diff
@@ -234,29 +234,26 @@ <h2 class="foot-section-h" id="contact">Get started</h2>
...
                      <label for="text">Your name</label>
-                    <input type="text" name="your name" placeholder="Anastasiia" maxlength='30' required id="text">
+                    <input type="text" name="your name" placeholder="Anastasiia" maxlength='30'  id="names">
                 </div>              
             
                 <div class="email-container">
-                    <!-- <div>Your email</div> -->
                     <label for="email">Your email</label>
-                    <input type="email" placeholder="|"  name="your email" id="email" required >
+                    <input type="email" placeholder="|"   name="your email" id="email" >
```
Comment (1GORDON) [inline index.html:247]: Great job @Eldonratzlaff 🤝

- [ ] You have done a great job. However, the form is not showing the error for uppercase letters in the email, neither is the form being submitted to formspree for lowercase emails.
form-error

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Eldonratzlaff ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] Kindly make sure your social handles in your readme file are customized correctly and the links work properly.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: popup.js [hunk 2]
```diff
@@ -203,4 +203,73 @@ function loadpages() {
...
+    const nameinput= names.value.trim();
+    const emailinput= email.value.trim();
+    const comentinput= coment.value.trim();   
+    if(nameinput===""){  const errornames = document.querySelector(".your-name-container");
+    names.placeholder="Please enter your name";
+    errornames.classList.add("error");
+       }
+    else { 
+   const succesnames = document.querySelector(".your-name-container");
+   succesnames.classList.add("succes");
```
Comment (brainconnect93) [inline popup.js:233]: - [ ] As requested by the previous reviewer, You haven't implemented the error message for the email address if a user enters an email with `uppercase`. Also, the form is not submitted to form spree when you have all your form filled with details

Your Form 👇 

col

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Eldonratzlaff, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Eldon, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- linters are passing

 ## Required Changes ♻️
- [ ] the same issues the previous CR mentioned still persist, there is not Error message visible when filling in an email with a capital letter.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @Eldonratzlaff, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- made changes ✔
- Linters ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**